src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(int firstLeg, int secondLeg) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public Figure createRandomFigure() {
+        return new RightTriangle(getRandomInt(), getRandomInt());
+    }
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. The `createRandomFigure` method should not be in the `Figure` class because creating a random figure is not conceptually related to the figure itself.